README.md

```diff
@@ -38,10 +38,22 @@ If you'd like to also classify nodes in ServiceNow's CMDB, then the module:
 
 ### Beginning with servicenow_cmdb_integration
 
-**Note:** If you're interested in both the ServiceNow CMDB external data and node classification features of this module, then you can just read the [classification](#ServiceNow-node-classification) section since that depends on the CMDB external data feature.
+Pre-reqs:
+  * Puppet Enterprise version 2019.7.0 or newer
+  * A ServiceNow instance (dev or enterprise)
+
+
+**Note** The trusted external data feature does not depend on the classification feature, so you can enable it on its own. However, if you’re also interested in classifying nodes in the CMDB, you must enable both features. If you're interested in both the ServiceNow CMDB external data and node classification features of this module, then you can just read the [classification](#ServiceNow-node-classification) section since that depends on the CMDB external data feature.
 
 #### ServiceNow CMDB as trusted external data
```

Comment:
Let's move the sections too like in the doc. So `trusted external data` should come after the `classification` section.

Reply:
I think you have an extra "trusted external data" section